Makefile

```diff
@@ -197,6 +197,11 @@ endif
 # On Windows only build tsh. On all other platforms build teleport, tctl,
 # and tsh.
 BINS_default = teleport tctl tsh tbot
+ifneq ($(CHECK_RUST),)
+ifneq ($(CHECK_CARGO),)
+BINS_default += gravitational-ssh-fdpass
```

Comment:
We need to run this always in CI or macos builds will fail. Do you know if this will always pass for both macos and linux in CI?

Reply:
I think that rust nowadays is mandatory to build the web ui assets, and i think that all the teleport binaries that we build have rdpclient (which means that the check for rust and cargo passes) but I'm not sure about CI 😬

Reply:
I'll test this prior to merge of all this work into branch/v15. Given the timeline, if it doesn't work then I might just update `BINS_default` to unconditionally include the new binary name.